components/frontend/src/dashboard/IssuesCard.js

```diff
@@ -0,0 +1,61 @@
+import "./IssuesCard.css"
+
+import { bool, func } from "prop-types"
+
+import { Card, Header, Label, Table } from "../semantic_ui_react_wrappers"
+import { reportPropType } from "../sharedPropTypes"
+import { capitalize, ISSUE_STATUS_COLORS } from "../utils"
+
+function issueStatuses(report) {
+    const statuses = { todo: 0, doing: 0, done: 0, unknown: 0 }
+    Object.values(report.subjects).forEach((subject) => {
+        Object.values(subject.metrics).forEach((metric) => {
+            let nrIssuesWithKnownStatus = 0
+            const issueStatus = metric.issue_status ?? []
+            issueStatus.forEach((issue) => {
+                if (issue.status_category) {
+                    statuses[issue.status_category] += 1
+                    nrIssuesWithKnownStatus += 1
+                }
+            })
+            const nrIssues = metric.issue_ids?.length ?? 0
+            statuses["unknown"] += nrIssues - nrIssuesWithKnownStatus
+        })
+    })
+    return statuses
+}
+issueStatuses.propTypes = {
+    report: reportPropType,
+}
+
+export function IssuesCard({ onClick, report, selected }) {
+    const statuses = issueStatuses(report)
+    const tableRows = Object.keys(statuses).map((status) => (
+        <Table.Row key={status}>
+            <Table.Cell>{capitalize(status)}</Table.Cell>
+            <Table.Cell textAlign="right">
+                <Label size="small" color={ISSUE_STATUS_COLORS[status]}>
+                    {statuses[status]}
+                </Label>
+            </Table.Cell>
+        </Table.Row>
+    ))
+    const color = selected ? "blue" : null
```

Comment:
Do we not have a 'selected color' preset, i.e. for tweaking dark/light mode differences? Or does this work for both

Reply:
Blue isn't actually a real color, but a class that gets changed by Semantic UI React to different shades of blue depending on the dark/light mode setting. I guess I could change `const color = ...` to `const colorClassName = ...`, but Semantic UI React calls these props `color` everywhere so that I don't think that's much of an improvement.